velox/vector/fuzzer/VectorFuzzer.cpp

```diff
@@ -922,6 +935,10 @@ RowTypePtr VectorFuzzer::randRowType(
   return velox::randRowType(rng_, scalarTypes, maxDepth);
 }
 
+size_t VectorFuzzer::randInRange(size_t min, size_t max) {
```

Comment:
can we leave this as a free anonymous function in the .cpp file (if you pass the rng)?

Reply:
I'm using this in my followup diff: D65926722
We wanted to be able to randomize the number of columns in a row vector based on the same rng seed.